css/main.css

```diff
@@ -0,0 +1,476 @@
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 50px;
+}
+
+.form-section-content {
+  padding: 100px;
+  width: 327px;
+  height: 216px;
+  display: flex;
+  justify-content: center;
+}
+
+.form-section-content-h3 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 24px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+input {
+  border: 0;
+  opacity: 0.5;
+  width: 327px;
+}
+
+.hr-form {
+  width: 100%;
+  height: 1px;
+  background: #dbd8d7;
+  opacity: 0.3;
+}
+
+.form-li {
+  height: 50px;
+  border-bottom: 1px solid #dbd8d7;
+}
+
+.form-textarea {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 44px;
+  color: #3c3a39;
+  background: #fbf8f7;
+  width: 100%;
+  border: 0;
+}
+
+form button {
+  background-color: #ff6b00;
+  border: 0;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #fff;
+  padding: 10px;
+  transition: all 0.5s ease-out;
+}
+
+.righthand-of-logo-2 {
+  display: none;
+}
+
+form button:hover {
+  color: #ff6b00;
+  background-color: #fff;
+  border: 1px solid #f56b00;
+}
+
+@media screen and (min-width: 768px) {
+  form ul {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 0;
+  }
+
+  .form-section {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .form-section-content {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    padding: 0 100px 100px 100px;
+    width: 550px;
+    height: 200px;
+    margin: 0;
+    margin-top: 220px;
+  }
+
+  .form-section-content-h3 {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    margin-top: 50px;
+    line-height: 44px;
+    text-align: start;
+    color: #172b4d;
+  }
+
+  .get-in-touch {
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .form-textarea {
+    height: 114px;
+    width: 684px;
+    margin: 0;
+  }
+
+  .form-li {
+    height: 50px;
+    border-bottom: 1px solid #dbd8d7;
+  }
+
+  .form-li input {
+    width: 684px;
+  }
+}
```

Comment:
🪛⚒️ [OPTIONAL] kyndly stick to the figma design,  in the image you can see the text is overlaping the form, you can prevent this if you use a section tag to wrap your contact me section, use display flex and flex direction as column.

![image](https://user-images.githubusercontent.com/95243996/190655194-9a4c7d12-25b5-41ea-9540-2c5cebd2a79d.png)